frontend/src/components/header/messages.js

```diff
@@ -38,16 +39,16 @@ export default defineMessages({
   },
   AuthorizeTitle: {
     id: 'signUp.modal.authorize',
-    defaultMessage: 'Have you finished the registration on OpenStreetMap?',
+    defaultMessage: `Have you finished the registration on ${TM_ORG_NAME}?`,
```

Comment:
Inserting env vars on react-intl messages will generate a mess, and TM_ORG_NAME here is the organization that is deploying the TM instance (on HOT case it is "Humanitarian OpenStreetMap Team"). In my opinion, we should continue using OpenStreetMap. When you are deploying TM in a context of another OSM server, you will have the OpenStreetMap website with the same name, wouldn't you?